tests/test_experiment_server.py

```diff
@@ -64,6 +65,17 @@ def test_consent(self):
         })
         assert 'Informed Consent Form' in resp.data
 
+    def test_experiment_status(self):
+        resp = self.app.get('/experiment_status')
+        assert resp.status_code == 200
+        data = json.loads(resp.data)
+        assert data.get('status', 'success')
+        state = data.get('state', {})
+        assert state.get('completed') is True
+        assert state.get('unfilled_networks') == 0
+        assert state.get('working_participants') == 0
+        assert state.get('nodes_remaining') == 0
+
```

Comment:
Yes, I definitely think a test of the actual functionality here makes sense, though it will require an updated example experiment that uses a non-Empty network. That's not a big deal, but one I wasn't quite up for at the hour I finished this.